src/modules/reserve-overview/ReserveFactorOverview.tsx

```diff
@@ -0,0 +1,83 @@
+import { ExternalLinkIcon } from '@heroicons/react/outline';
+import { Trans } from '@lingui/macro';
+import { Box, SvgIcon, Typography } from '@mui/material';
+import { ReserveFactorTooltip } from 'src/components/infoTooltips/ReserveFactorTooltip';
+import { FormattedNumber } from 'src/components/primitives/FormattedNumber';
+import { Link } from 'src/components/primitives/Link';
+import { ReserveOverviewBox } from 'src/components/ReserveOverviewBox';
+import { ExplorerLinkBuilderProps } from 'src/ui-config/networksConfig';
+
+interface ReserveFactorOverviewProps {
+  collectorContract: string;
+  explorerLinkBuilder: (props: ExplorerLinkBuilderProps) => string;
+  reserveFactor: string;
+}
+
+export const ReserveFactorOverview = ({
+  collectorContract,
+  explorerLinkBuilder,
+  reserveFactor,
+}: ReserveFactorOverviewProps) => {
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexWrap: 'wrap',
+        justifyContent: 'space-between',
+      }}
+    >
+      <ReserveOverviewBox
+        title={
+          <ReserveFactorTooltip
+            text={<Trans>Reserve factor</Trans>}
+            key="res_factor"
+            variant="description"
+            collectorLink={
+              collectorContract
+                ? explorerLinkBuilder({
+                    address: collectorContract,
+                  })
+                : undefined
+            }
+          />
+        }
+      >
+        <FormattedNumber value={reserveFactor} percent variant="secondary14" visibleDecimals={2} />
+      </ReserveOverviewBox>
+
+      <ReserveOverviewBox
+        title={
+          <Typography variant="description">
+            <Trans>Collector Contract</Trans>
+          </Typography>
+        }
+      >
+        <Link
+          href={explorerLinkBuilder({
+            address: collectorContract,
+          })}
+          target="_blank"
```

Comment:
Suppose these uses should be changed to `isExternal` now..

Reply:
Disregard this, but in response to your comment, yeah a prop may seem redundant when passing in a link elsewhere, but is more robust than the current check.

Reply:
@defispartan Last ref here

Reply:
Fixed